SharedPackages/BrowserServicesKit/Sources/BrowserServicesKit/PrivacyConfig/Features/PrivacyFeature.swift

```diff
@@ -72,6 +72,7 @@ public enum PrivacyFeature: String {
     case setAsDefaultAndAddToDock
     case contentScopeExperiments
     case extendedOnboarding
+    case visualRefresh
```

Comment:
Please bear in mind that this enum is common for iOS + macOS.

We already have a flag for this, defined for the iOS project – it’s called `experimentalBrowserTheming` and it’s just a few lines above. I think we could reuse that flag – the changes in Privacy Config will be separate for iOS and macOS anyway.

Reply:
Aha, good idea! Didn't know about `experimentalBrowserTheming`, let me use that one.